### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by k-takata**
Update for #2209.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -80,6 +80,9 @@ man/ctags-incompatibilities.7.rst
 man/ctags-optlib.7
 man/ctags-optlib.7.html
 man/ctags-optlib.7.rst
+man/tags.5
+man/tags.5.html
+man/tags.5.rst
```
Comment (masatake) [inline .gitignore]: tags.5.rst was added to repository.
Unlike the other man pages, tags.5.rst is not generated from tags.5.rst.in.
So man/tags.5.rst should not be ignored.

Comment (k-takata) [reply .gitignore]: Oh, I will fix it.

**Comment by masatake** [comment]
LGTM.

**Comment by masatake** [comment]
Thank you.

**Pull request merged**